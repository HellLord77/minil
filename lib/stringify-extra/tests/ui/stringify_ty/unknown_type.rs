use stringify_extra::stringify_ty;

fn main() {
    stringify_ty!(byte);
}
