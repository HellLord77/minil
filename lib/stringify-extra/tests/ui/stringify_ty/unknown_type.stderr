error[E0412]: cannot find type `byte` in this scope
 --> tests/ui/stringify_ty/unknown_type.rs:4:19
  |
4 |     stringify_ty!(byte);
  |                   ^^^^
  |                   |
  |                   not found in this scope
  |                   help: perhaps you intended to use this type: `u8`
